api/envoy/config/rbac/v4alpha/rbac.proto

```diff
@@ -66,19 +66,28 @@ message RBAC {
 
   // Should we do safe-list or block-list style access control?
   enum Action {
-    // The policies grant access to principals. The rest is denied. This is safe-list style
+    // The policies grant access to principals. The rest are denied. This is safe-list style
     // access control. This is the default type.
     ALLOW = 0;
 
-    // The policies deny access to principals. The rest is allowed. This is block-list style
+    // The policies deny access to principals. The rest are allowed. This is block-list style
     // access control.
     DENY = 1;
+
+    // The policies set the "envoy.log" dynamic metadata key based on if requests match
```

Comment:
Does this still effectively allow?

Reply:
It allows all requests. I'll make that clearer in the comment.